lib/sisimai/bite/email/aol.rb

```diff
@@ -0,0 +1,208 @@
+module Sisimai::Bite::Email
+  module Aol
+    # Sisimai::Bite::Email::Aol parses a bounce email which created by Aol Mail.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Aol.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->